install/jakartaee/docs/index.html

```diff
@@ -36,22 +36,24 @@
 </head>
   <body alink="#0000ff" bgcolor="#ffffdd" link="#0000ff" text="#000000" vlink="#840084">
 <div align="Center">
-<h1>Welcome to the Java EE 8 CTS<br>
+<h1>Welcome to the Jakarta EE 8 Platform CTS<br>
 <span class="emphasize">Your Starting Point</span></h1>
 </div>
 <hr>
-<h2>Guide to the Java EE 8 CTS Documentation</h2>
-<p>The Java EE 8 CTS documentation includes the following:</p>
+<h2>Guide to the Jakarta EE 8 Platform  8 CTS Documentation</h2>
+<p>The Jakarta EE 8 Platform  8 CTS documentation includes the following:</p>
 <ul>
-  <li><p>The <em>Java EE 8 CTS Release Notes</em> provides information about the test harness, the test suite, and the platforms tested.</p>
-<p>The Release Notes are available from Java Partner Engineering.</p>
+  <li>
+<p>The <em><a href="../../../internal/docs/el/ReleaseNotes.html">Jakarta EE 8 Platform   CTS Release Notes</a></em> provides information about the test harness, the test suite, and the platforms tested.</p>
+<p>The Release Notes may be udpated and released through the Jakarta EE 8 Platform Specification web site:  <a href="https://jakarta.ee/specifications/full-profile/8.0/">https://jakarta.ee/specifications/full-profile/8.0/</a><a href="https://jakarta.ee/specifications/full-profile/8.0/"></a> or <a href="https://jakarta.ee/specifications/web-profile/8.0/">https://jakarta.ee/specifications/webl-profile/8.0/</a>.</p>
   </li>
   <li><p>The <em>Java Platform Enterprise Edition 8 Compatibility Test Suite User's Guide</em> provides comprehensive user documentation, including instructions for installing and configuring the test suite, and using the JavaTest test harness to execute the tests. The User's Guide also contains the rules you need to pass for certification.</p></li>
-<li><p>The <a href="./assertions/JavaEESpecAssertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Java EE 8 CTS test suite.</p></li>
+<li><p>The <a href="./assertions/JavaEESpecAssertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Jakarta EE 8 Platform  8 CTS test suite.</p></li>
```

Comment:
The file name shouldn't have "JavaEE" in it.

Reply:
will fix